mkdocs.yml

```diff
@@ -62,33 +58,17 @@ color: "#82BDC3"
 
 nav:
   - "General":
-    - "About": "about.md"
-    - "Architecture": "architecture.md"
-
-  - "Quick Start Guide":
-    - "Download": "download.md"
-    - "Quick Configuration": "quick-config.md"
-    - "Run Iguana": "run-iguana.md"
+    - "About": "README.md"
```

Comment:
Yes, should be possible, but also difficult, because the file probably doesn't change that often and you'd have to check for each deployment if the file has changed or not.

Reply:
nevermind, added it now